feed/tests/test_post_model.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from feed.models import Post
+import django.contrib.auth
+
+
+User = django.contrib.auth.get_user_model()
+POST_TITLE = "Some First Post Title"
+POST_CONTENT = "Some first post content here"
+
+
+@pytest.fixture
+def users(db):
+    user0 = User.objects.create_user('user1', password='pass123')
+    user1 = User.objects.create_user('user2', password='pass321')
+    return [user0, user1]
+
+
+@pytest.fixture
+def post0(db, users):
+    return Post.posts.create(title=POST_TITLE, content=POST_CONTENT, author=users[0])
+
+
+@pytest.mark.django_db
+class TestPostModel:
+    # Testing different post attributes
+
+    def test_new_post(self, users, post0):
+        # Testing post data
+        assert post0.title == POST_TITLE
+        assert post0.content == POST_CONTENT
+        assert post0.author == users[0]
+
+    def test_post_save(self, post0):
+        # Testing post save in DB
+        post0.save()
+        assert post0 in Post.posts.main_feed()
+
+    def test_post_like(self, users, post0):
+        # Testing likes DB
+        post0.save()
+        post0.likes.add(users[1])
```

Comment:
once you find the posts add the user returned

Reply:
Done